randovania/games/prime2/pickup_database/pickup-database.json

```diff
@@ -513,7 +522,8 @@
             "offworld_models": {
                 "am2r": "sItemSpaceJump",
                 "prime1": "Space Jump Boots",
-                "dread": "powerup_spacejump"
+                "dread": "powerup_spacejump",
+                "samus_returns": "powerup_highjumpboots"
```

Comment:
probably better to use SJ here like dread does

Reply:
I specifically chose hj because it's is boot. SJ (while similar in name), felt weird to use. If it makes more sense I can change it.

Reply:
I'll let dunc decide on this then

Reply:
i prefer space jump